packages/frontend/src/hooks/useSpaceManagement.ts

```diff
@@ -0,0 +1,67 @@
+import { useCallback, useState } from 'react';
+import { useCreateMutation } from './useSpaces';
+
+type UseSpaceManagementProps = {
+    projectUuid?: string;
+    defaultSpaceUuid?: string;
```

Comment:
in what cases do we default to a space?

Reply:
When the user wants to create a chart from space view (see 'fromSpace' param usage)